securedrop/journalist_app/__init__.py

```diff
@@ -131,24 +131,25 @@ def setup_g() -> 'Optional[Response]':
                 flash(gettext('You have been logged out due to password change'),
                       'error')
 
-        session['expires'] = datetime.utcnow() + \
+        session['expires'] = datetime.now(timezone.utc) + \
             timedelta(minutes=getattr(config,
                                       'SESSION_EXPIRATION_MINUTES',
                                       120))
 
         uid = session.get('uid', None)
         if uid:
-            g.user = Journalist.query.get(uid)
+            g.user = Journalist.query.get(uid)  # pylint: disable=assigning-non-slot
 
         i18n.set_locale(config)
 
-        if app.instance_config.organization_name:
-            g.organization_name = app.instance_config.organization_name
+        if InstanceConfig.get_default().organization_name:
+            g.organization_name = \
+                InstanceConfig.get_default().organization_name  # pylint: disable=assigning-non-slot
         else:
-            g.organization_name = gettext('SecureDrop')
+            g.organization_name = gettext('SecureDrop')  # pylint: disable=assigning-non-slot
```

Comment:
Nit in passing:  I wondered if this could be simplified, since [`InstanceConfig.organization_name` has `default="SecureDrop"`](https://github.com/freedomofpress/securedrop/blob/develop/securedrop/models.py#L872), but it looks like that default's not applied in the [upgrade migration](https://github.com/freedomofpress/securedrop/blob/f768a1a5aa37e790077dfd38d6e7a0299a4b3a0f/securedrop/alembic/versions/92fba0be98e9_added_organization_name_field_in_.py#L19-L24).